ios/engine/KMEI/KeymanEngine/Classes/KMManager.m

```diff
@@ -1,4099 +0,0 @@
-//
-//  KMManager.m
-//  Keyman Engine
-//
-//  Created by Patrick Weekes on 12-05-16.
-//  Updated by Serkan Kurt on 17/05/2013.
-//  Copyright (c) 2017 SIL International. All rights reserved.
-//
-
-#import "KMManager.h"
-#import "KMManager+Internal.h"
-#import "KMInputViewController.h"
-#import "KMURLProtocol.h"
-#import "KMTextField.h"
-#import "KMTextView.h"
-#import "KMPopoverView.h"
-#import "KMKeyPreviewView.h"
-#import "KMSubKeysView.h"
-#import "KMKeyboardMenuView.h"
-#import "KeyboardPickerViewController.h"
-#import "UIView+Helpers.h"
-#import <QuartzCore/QuartzCore.h>
-#import <CoreText/CoreText.h>
-#import <CommonCrypto/CommonDigest.h>
-#import "UIButton+Helpers.h"
-#import <objc/message.h>
-
-#pragma mark - Constants
-
-static BOOL enableWebKit = YES;
-
-// Keyman system-wide keyboard
-static BOOL kIsKeymanSystemKeyboard = NO;
-
-// Group Identifier
-static NSString *applicationGroupIdentifier = @"";
-
-// Notifications
-NSString *const kKeymanLanguagesUpdatedNotification = @"KeymanLanguagesUpdatedNotification";
-NSString *const kKeymanLanguagesDownloadFailedNotification = @"kKeymanLanguagesDownloadFailedNotification";
-NSString *const kKeymanKeyboardDownloadStartedNotification = @"kKeymanKeyboardDownloadStartedNotification";
```

Comment:
The "notifications" section here was moved to NSNotification.Name+Notifications.swift.  Just a bookkeeping note.

Reply:
And as another note, this was done because these constants were bridged to properties of `NSNotification.Name` before converting `KMManager`.